patches/net/minecraft/world/level/chunk/LevelChunk.java.patch

```diff
@@ -9,6 +9,14 @@
      static final Logger LOGGER = LogUtils.getLogger();
      private static final TickingBlockEntity NULL_TICKER = new TickingBlockEntity() {
          @Override
+@@ -137,6 +_,7 @@
+         this.setAllStarts(p_196851_.getAllStarts());
+         this.setAllReferences(p_196851_.getAllReferences());
+ 
++        net.neoforged.neoforge.attachment.AttachmentInternals.copyAttachments(p_196851_.getAttachmentHolder(), this.getAttachmentHolder(), type -> true);
```

Comment:
It would also be good to have a test for this stuff, but that might be too hard to write.

Reply:
It is hard to write. I've created a custom feature that sets an attachment in ProtoChunks and then observed the checked the attachment in level chunks, but I have no idea how to write an automated test for it in a reasonable way